3.15.0- save files unloadable

It seems that changes to TradeOffer in #4127 broke compatibility with old save files.
I suspect the issue reported in #4155 might be related to this.

Was trying to ping you on discord @SomeTroglodyte, do you happen to use it? 


Here is the excerpt of the stack trace:
```
com.badlogic.gdx.utils.SerializationException: Error reading file: SaveFiles/Route Quest debug
	at com.badlogic.gdx.utils.Json.fromJson(Json.java:797)
	at com.unciv.logic.GameSaver.loadGameFromFile(GameSaver.kt:58)
	at com.unciv.logic.GameSaver.loadGameByName(GameSaver.kt:55)
	at com.unciv.logic.GameSaver.loadGameByName$default(GameSaver.kt:54)
	at com.unciv.ui.saves.LoadGameScreen$1$1.invoke(LoadGameScreen.kt:47)
	at com.unciv.ui.saves.LoadGameScreen$1$1.invoke(LoadGameScreen.kt:23)
	at kotlin.concurrent.ThreadsKt$thread$thread$1.run(Thread.kt:30)
Caused by: com.badlogic.gdx.utils.SerializationException: Class cannot be created (missing no-arg constructor): com.unciv.logic.trade.TradeOffer
Serialization trace:
theirOffers (com.unciv.logic.trade.Trade)
trades (com.unciv.logic.civilization.diplomacy.DiplomacyManager)
diplomacy (com.unciv.logic.civilization.CivilizationInfo)
civilizations (com.unciv.logic.GameInfo)
	at com.badlogic.gdx.utils.Json.newInstance(Json.java:1220)
	at com.badlogic.gdx.utils.Json.readValue(Json.java:1008)
	at com.badlogic.gdx.utils.Json.readValue(Json.java:1108)
	at com.badlogic.gdx.utils.Json.readFields(Json.java:901)
	at com.badlogic.gdx.utils.Json.readValue(Json.java:1073)
	at com.badlogic.gdx.utils.Json.readValue(Json.java:1108)
	at com.badlogic.gdx.utils.Json.readFields(Json.java:901)
	at com.badlogic.gdx.utils.Json.readValue(Json.java:1073)
	at com.badlogic.gdx.utils.Json.readValue(Json.java:1068)
	at com.badlogic.gdx.utils.Json.readFields(Json.java:901)
	at com.badlogic.gdx.utils.Json.readValue(Json.java:1073)
	at com.badlogic.gdx.utils.Json.readValue(Json.java:1108)
	at com.badlogic.gdx.utils.Json.readFields(Json.java:901)
	at com.badlogic.gdx.utils.Json.readValue(Json.java:1073)
	at com.badlogic.gdx.utils.Json.fromJson(Json.java:795)
	... 6 more
Caused by: kotlin.UninitializedPropertyAccessException: lateinit property gameInfo has not been initialized
	at com.unciv.UncivGame.getGameInfo(UncivGame.kt:39)
	at com.unciv.logic.trade.TradeOffer.<init>(TradeOffer.kt:15)
	at com.unciv.logic.trade.TradeOffer.<init>(TradeOffer.kt:10)
	at com.unciv.logic.trade.TradeOffer.<init>(TradeOffer.kt:24)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
	at java.base/java.lang.Class.newInstance(Class.java:584)
	at com.badlogic.gdx.utils.reflect.ClassReflection.newInstance(ClassReflection.java:93)
	at com.badlogic.gdx.utils.Json.newInstance(Json.java:1202)
	... 20 more
```

Here the save file
<details>
  <summary>Save file</summary>
  H4sIAAAAAAAAAO29a3MjR5It+Fe48+V+Qcjy/eC3UnXrsaPqqlFVm/ZemWwtCSRJXJEABwAlsWX13zcT+fDjER6ZDpCsnrnb3TIJBCIzI+Ph7uF+jvufy/Vv67v1P6rDervZX/785832bnVpkmTR/PC36r6+/LraXVW7dbXZLw718vbyz/bf+x/rfV3tlrf16vLnT7f19m5787R4Vx+qu4u31f6w3tws3jY3POwel+2dF2+ru3qzqnaLd9Xhtr5vHrfcN5/3283uafHddrevr6rlrxc/rlftpe/Wm/Y/f93crDd1vWs/f9pVDw/th693280/6ouftrtfj9/fNp9v6/pu8abtTnO3D9vDof3vm836vunOd4/7q2qzar74abc+duzNzW69fLw7PO7qxffN3cZ7fbhd323324fbp18+Lx62d+vlut5f/vl58Z+P9f7wrtpUN/Wu/bsdpHrz5qb9dX/Y7urVd8fe1fv9ZR4Hi83j/VW9e3/9LbWLmqt2dXX4UG8f7urLP/s/ds0IfNiuN4f2OZ8Xv62Xze2e4FEP24fHhzd39e7Qzs7h6aG+HO+6+K26e6wv/+3fPi+8v/yyqP94uGu7+Gl9V7f3+OPSxIunS9P0qPkcBu3n/Pi5aD9mx48maT/Hx8/dv/Pmi/T4KWs+JXQxXdtdGkbNx6K7S3epoR8NXNLe0XS3NHR3Q7eHbh67HFHXuhtDJwt6jWy8Vw4vVIzPNRl7laEL6fjcHLp7fEQYUROTwIv0r3l8ZROG7JoY2pXd565Z87n/zx/9A9vvw7FDMcxANL5nNL4mjH3X9eMDcxr8fh6ysYPjc//ouj88zdCDC3q1bkmkNMIw7hGNIL5fBhdm4/1S+jaj+Uioozl1v6Be0NwlMOzd4+Dj8euYlk8yDmDXyWxcQyEtIUNvG1M/EhqwjLqUUoNh0um6AvYHjC49LoORo0517xKNb0h7y8CrwLvCZbBhE+olTDYsga7v6Xgf3OwwAf1YF/AN7axhwdDrwUdDLY7X0ajQQh06T9/m1A14cxqOiMYgoveCVY4rdJjAcbihO4N0i2g5QN+hcUBDHdJQh/TEmJ6Ig4jTGY73QAkw7I+MjcnQgnaJgYf1T0jgNjQ4IY1YSOPUfSzbT/0TSxA/Ie3wMGw/9sLu+IcJcYmHeHVCd03YFRncCz/DzA4vFPDepXh/9kdGjTL8PqfvcxhM6GgnV6EzzXsz4YuzzpqFILFDfAJ+DlH493dO6JKCOtKvo27K4KWhryHMc8huyeckhlFlMxqxZjBDEVwB3YvgjaKxda8kj60LahxDY3gqjsSg7FI2wrhEYntQce2U1IWCtY+ofQo3hSVS0NeG+ky67TirMLl0yfhr3zDvGubwK62ylEbG0JrsZeTxtfuHxyRsQxLB4ztGsDiggRlawK+0dOBjSR04PqqgR1Fn6aEhrm9YcLjUYXUbGA4aetCWdGfTjyuspAxGDkyzEK6HkRl0TUodjvlAgwEUdw8jpRriJMbwOaH+9BMKt4Efh7XOp6uf0NLqX079o8lP6LKUHpF2D02hS/RrQf2HcTO0MRO6zMAbFjCaNC8GXqcz3bp2CQizBKRoYs0ASJNBYaZWH2jRdouzpBeAQS+hezF9HgRRbA99Or4CDFwAIxbADWlwDIiAfpgDfBrMHbxOSnNHcjGm53Y2d7854oU1d2i7wff96u//A7sCRF//MYMZyGBmUhgV2smd7RSN3TOwqTJ4WWg9SEm4Eu+dWjNBopG6Db02rNs5PLLEH9jzwS6M8aFkY4YgKGHZgXYarCZocnxsCcMyXDm+Cyw8M6yFBN6dDhLdDAbH6cqxjxn1EeQULMZeUMCtEhjR8Wljw/6+Od2XXQaLNKKV2b0ZqcGIukLm9bAA4WsD35N0QlMWDmgFLDzcOKCCDEmXqHspkp4oFxJcC2Blg5E+jFBBQ5vD1BQw+mz+x0cPEgNHFebYkLAraKyD8UsD51vYZrCfWE/IVTGaeHQ46tsU+NZspeN5qKTe0HFvaJ7DIsitjTHuAHqHEqYMZxX8ALgHcuwgHXpD8l305lCKvYEtwI4uIFZw3Ps/BkOH5DKcGumcafC+ZBxBWwON8WE5SiBmWOYwGzgUIayVQViAcyiDsc9o7FNrQA1uuXDsLRz2UGwbkqy4QTqDBQ/u/WtltDRTmrKE9iHcFA7dhrpg4Fl0yjdkpxxXK278cYdH43xzH43huxLOlQEMXYGjTxsNng/OwhAHvcS78wejYQ0rvISBwzMqTAJsIvLchOAfYdsVVzCuKTLJaNuRDxJlG4gW2Ge4WkMS2wZkNaztHAciYQMBfjLwAAzmTwCCKYNFWo6DksJAZ9ZmgPMVilOSVGyCmHAgX0z/MbMGPB0/DhL38yDT4fBhaCO1v8NOGaReBEuvmybcZcMu6BujCsalMzwVfSPDbOV8/Ituhw53DKwtSkIRncAwlaT/QQQY2KJkYw4PIY+TAX8iOIcKmMN+QIbdYrUq6fVwmbP1FcCrkYsgwqEL+BrHoQJrAc4SJDINHSpSEDsovMlxy74uWb9ATqPvClbkIADAX9u/Ba29ECUDrdny8y/N54e76qnefWrDIN893lebY5DmMsuDMZL1/W77n4/b9RDH2i/X9WZZv7/+odofik+Puzb+leWLLHH+yX5ZOFEvHuwaY2D6qNfHQxu1+nD7tG+//S8YA+ve+dN2eOvLn7993Dxsf1/Vu/637zcfdtubXRv2+nP87bKMo88YP+siZG+7p13GZbGoVtuHQ736MDT5+bvtZrtbvN/dVJv1P+rVxY/13fqmHcF3zZsftstdtWyGtun6P+rN/nb9sPiwrg9Pix/WV/Wu+W87+n2b7d1d3Qz+b/XFj4939S9jKO6N9cjitKhekudSVC+0onoPx1DeN9vd3+o/Dt9SiO8yCYKF++O39abeVXeXURwshIDgw2676lbSZZRk3YIOg2jRT9xlnvSP7+/TX9fuezeQuFo/3G3vmzG6/PPd+q7aXP65bdbk7m2/OY7fDY2adfrxUB0e95cf6mpZL67vqpv92+3j5rDa/t5c+dff6t3Xdb35ZtdsodX+uKfp0nfb1fp63bzJ5Z//s1k4+/fXx7tcNi/58bZqhvWvm/q+FSifF+/+x9fV5h+V+fft5mZr9wh/e+2eNX35j8f6ql5eNKvsyeoJ/PLFR+jNrrpaV1aHui8Xh121OoZ0m9/WzcK8bu/485/Lu6pZscvt/VePm0b2fdXIpfXyq2Pjrz61/z62XGzaO/1024ioRiK2YvOHxz8ed0//b7Pdt4+75p1Wj7sjLOAybCTsovnurCd8W99r7t9I8PPf4vvfmrU+85Aoe8ZLfFzfNROqeELzDM8KUa7CVgI48919ef6do/SrRouuN9d3j63Ka9R6o6rfbA6/17sHe2l13z7zLRq508gTe0Mfv/zSO+jzYrM9NG2XI8Tl0Ijfy3/7+f1mu22U8031y8Vttb+42TWP/b/+bbFs9HgjMG8ak7Nq5XvVCWDPckH4zFc/bLuHvOm0/92Wnkk+lV8+t0v92IU3m4u67ebFz2+rQ/XQSPWuK9Xh0BgCjRr8+f2+tQJ+rX4Z+7V78X4N7kXoWGsMHHuyP1S7RmteNAbCzz80cvh2+7ivsTPBV2mSBmUR5Iub5o8oD9KiiNLFVfNHkB9/Or2/raDt+7o8SuPB8s0+Ux9/blbPVaPhj/28ahbKxdXj+u5wsW66+nXzsF316v2cHte+xzPz/Xs7yg+tubZqu96IlIvGcmuNnt3TK0764D1qetfYSM02bMyHn5rtcBSN7+9WF99U68Pt9ePd4sdtY5Nury++XV81bZr18Es7KUer7c/hppewvhfr1eX1srzOk6wwyyAsTJIXlbm6Xi2bf12vqjrK06y66gTruAkX183Wbw3fRlqApd5Y5W+W//m4bnZ0Mwa3ddOD28YYCo64qLv+4fC5OT9cb7erj0fzrbH+F/uHermu7tb7I8IK/nxzR0Py52BlDyP7v6vfqq8ag+jmq+83h7qxoHpAVdguwHZNopnfXH9cel83/2rfoBmZd9vGWKw3h8WH6q4Vbt1aXRwFXt2O5v+qH/dHu/3D0666X6/2reW63e/rx/vFm8Ywbe2+xU9VsxQuGsvsbjFuvtbm3VWNtvt219h2zX8/1feN6dne802zVe+bQfupurvb/9KI+8E4tzr7dbVrbOVf95dpcyhcwm+NndO848/D741Orv94qDb7bsl1Rmc/skXRjUP7RseBPRC6zD4NS2d8O45iwBNNf4FbjDV2XXPMmYDusNj+7PoxMV7VXZs2m+L3Rga6oLnYfTK+CvZ0tkf0M3YmnuvYev9+1xyONtXd2+ph3RxCL5v5q5tGbDeOQaNmO6ZZI+riIjZVFacmSZKVqdJqZYLwOiyzKgzS8Lrfjr2+UezGeNyOYZH7tmOBu7EIrN2o20rV43571+yLdol/11y7rHabZo82+2fcZZ/qdsU2QrRqNsH90+K7qt3PNxffVrvmpLZv9t120x2XcZ8N65w23PFQdtHun35bTey7uV3Wbu73zf2bY2EYFeJGoyZTW60M/VuNoRMRTWGYpw1DOCycZxhahoU5UvcaXMzMaYyhOMPjtQHejT2UwctCvEEobT/WBf8d/G+XePtDytDaRD2ortlCV3ESZ3m5MldxlJgkWjabKVqFZhXFWZkGeVjlvUZr18fq4sd1e16Y30VpMSq1VKnUwuifrtXGDaPbJcKem9g4f9sedvXFX9qxjNNc3DjUZFJHpf6NQ65ZQzBZDK/k/CNgkyFYDZ54aAuxkoIcnHAH8DWD1zWXVr3cz5zfJBd7VDo9KvhlhaJH0o5olVerVaL4enUdRaYqskarBElirtLqyuTLPA6vo7wui6LbEp05znZDf6Ll9l2a+HZAwjbAOWqkt8Cs9fmLtLrooDOxuJLptTUoe4xNYJwBHfE2CJ1di58xCFBIqwWf4DyZ39WZ2QEm2Aq7KIuW1dXSlHF5bZI4T5s5LlJznUf1Vd7MeR6V3cy++fX3el/t72uN+d7IOs/0cvkWnzO/zsR65csojI6vK8z/2GBq/ts4nW8BIPoS4UcImcRYVg98wyvwc+xMNYeWofJsdac1s4Ot2kxsVgTNJksTU0fXS5PU+dIUy+vaJFmUh6t20pN+y7b7s25d+Pv5bZulCh3Gpzg7yxJ01I+tbY523m29ualnzTMwCY/hTWEdQJPJlRD6DTQ4JyChhj7CIQE2L8A04bwTWR/hZhSmMxCedhBmcM6AYwZwKuDAkVgNoJORIHvk5oPPy3doGXwJb1vDu5WqM2wlNNhoWGkg0FylSDZhlJAM5BKSQK8CNwkeBtwkmYXkYSwBrMRmSxnsFjyV061oYG2GEtCagAKFusNWIy4fKrXecDx+EsUJv+YdGi6kzjFmVACKq/tjiAyP3AeKCXspT8i/QUQFHuZlQhVQlWQuU0azN9z9CZkkANnAYRmP5bTOHU4VsKTgKQZfCMhRwH0iQ83PgkKYFC0KHCgwBFzOFT3XwIM99Kowt7rPzlfjyu/fenx3PH3CXA43BXgesLYsGebwqga5SFdDh/D+0GfiVCH/iuB4QFRjeJ1h5OmJIB0Q4pTQI6e4W/1Ajn+My7KH2sAuQLAIjR3Dd8A74/eIlaN171C1+k1JMCgA0wDRCQdzAH7iEMHviLJhJ+txYhHYMcXK6nk740KhvrMBGLpJvcUuMNQPQKZhHbh0LZkIZhi9x7BdARMFTEp8jwjfqf+jg2kNchGgYDDtjJAFpznWbR09rO/4iKyCUWRPGd4M2WEG6WEuz2yGKTb8BWJnHFPnJVHAsh8EWhiyuY7fD3oFvuf0Lz93TOB+dY8Cao5A9urahOw9hxHkzC7c0kgA8xO7DHLP4EDBKWkLfhIYHk/MLQ23C57KSFv0GThYwN96NmNrUJaMvQWXGOsaTt4KadKBpkU8LgOUrqHpE2Ne9StAZHMZuCfwueiyftwc8hZ+jfyuZguT7ED2oWFURInkZZAQtrAOjw7hy6ZmGeQ8wLDj4kJ2QB+aENhddDTlyxGudmhghhCOtD6B+WWQJ6angeEdGNhxgeBzhwv2xKhGsJCQEDhcBPyN0n4aUjZhucFrOCQz5ImNLDP6xrqTYcPD2WV8g7jMMgwEIA8NGWcOZwyiU0hNcghqU6S0/q5IKGIT1q8CIpwZgVVGZDMgsGEDIrPxxAqcx4bcMx/hjDHYWLQNIxlw8Xgt3+4OZ80Kggxfwl5FOqPLPxuZbNy7Y4Cq4JLZ4LYgahiVjHhqTKAIQVnLa0D7GlzlVkRokqEGXwMTDvrmIcLBPSTuHfOOi1Q2ZKkBH42kL555GEsNQ0nAbwOWGtDT5iltBp/OolkjA41kEhDcgJ8mct0YbQ35bkBbA6IaRqFxVbPA2CAZkU0gxNxgsTK7Z1iF3bKkJYkqxeW2WTQ3ID+g0czYS4xFhHua1ityZG02GcIT4DQ6RZITaHcyJw2BB/BE6hJCI+jhAIaA3gxrtvsP0upYbBd6Ah10GXlssYQ4hhieQC4I7ALWiYKeMAhBPPoOXwLPDaJU8BHvjDw5JMIhyUwI5jpMNmwvsuUscW+z2sguZMQ7l76GdDto6aG/wW0Z+w2Za4z91m+qYUiHnYa+DhbOduLhAmUOvoZvgTCHZDBEryB5DClgnGImkNCAOmYHLcF77dDjgAgnMeXGbyRuHnL6gMSGd7eCqnxeXV4cdBzfJ4cRB4Iap79ZzDhgNAHrDal0wHqDeK7d2CXRMQiRjxiXWG9C71dgbyX6GlDWbAqehyjnBJ2RYEc98XDqclhy8BB4xtA/YCaSScTkm8N0Gy4dthrmLQrZUKBxjpMF26ikProUPAOdAToetCYhjrNHu8OFuoHmBcVLjrJhrcDGYO4CeA1keJKXExc4hhmI9cV8+yOn/jPR8yTqHuPfSfw96ig9Q2bZyYw8BN+JnD5Ow2PeQJGFZ9PluDycZ+GxiD9E821GH+o0kIvQf4edh/gTaGyR9sjgRUoeJ9lx6h7635GX5zL32pDzkZ2URvFItxtIKkey3fWurj+1lLEWvSER7/J0kZWLPDz+E43/CMS79w89jY4odd+2QeK9RJn7WK3vjt+tHruI+EiRe/u42zXdAK6cjxJ3Yn7Ljt33dnv/UO0FXp+CxOcj7b1t8d0XH+vdb+tlPc3UO5ILBZbe8fvLJAsmGHplnrsMPSTfNe92X7fUl4F/Z7HuFj/WD49XzZV++l16Gv0ulMh38Sz5buDXhTK/rl+Il2ko8ujy+KV5dF+CiaZl6DyXefcCfKYX4K9938zfZl/PMbOS53C/HprdNkdha5/g537Z3KbvbzbN+m52/P3H+nBoBdTfmoH7e7MtCiXJ6cf6+nFfrz5t/7Y9HO9Rvz0CKfr7mOSrI3lQSfHyPcgljQ3QLuuWhPh6/pQqCYNnz6ea8vj5OZRENWPw7PfQkh6fTUk8g8ynu/Nnh3mkJhuNEd4W1baMszpNcpOt4qVJsqvAXJVhYFZFUWfX5TK5vk66IXtXL5cchzoYKYzZoOAZcWJD8iw4W88R+rr6R9UYECNQ9eb2MIWpViCpO1PnMs4XrZHRfMoSEejWtZvCuEXBBNrRBiOMljtamBFaovgDi3NIbliWNsaN7Bt+jhv+wC8T9+SGrsTeWS7gaUvPm7FXHp/lfU/p1bQ8niFa3KzzYFmGVZDmpoiClUnKq6W5SqLmX0WQhNlqFZTxuM6bNTa/0P1ozRyXd3wW4Hpc3m/Xu+Xjfljewro+Iixvtw/jAtcs7R4CepnI1Jrh90kIdzSxqCFIL6TjZWCCEW0A4BoGVhk8o4AIZHlOEGiCqWCwrbg+x9aJcBXe1dez7o1cLPgAv2lWXRyvqlURxaaIryPTLLSlKcuiWYRxXOfLIA+Wy1W36v5S3Vf7dqpn113sW3clE6vPWnb+9daetDqmly1Z9TLVAx2fl6QTYGEL9YS4U4Yd4mgSBthk8CQG+UmdP5gfmTnvWWQswWYM+ZTgH/L6dHAvcufcV029nYuknvadcxZyv9abZbzM06sgXy5NtrwqTRKkqSnCJDFRXadVHtVllIQDW+Xmtlnys6u4CBWw9+dZBiPQXWav9JYAshtPWMFJv0Uucxn+3v36LA4EeeUgiA0OSAyC07fgxocGbjQMAd/9N5+7uJm0FPktQrETEgXwuHpWZV3lq6Qyq+syNklVpi2DtmyW0DLI0jypszjqVs/fqv/dTM7s4im9lFmmetPzKLOWDPRxn7xLhITmZZ6Ji4NaTJqN2QRLCtyiLgAU5V3Cv4aIHqBMAYEr4DcH9yY1Al8mhHHBYwzAVlG2pW5HGfbSuX9mv4ardsegdrPkkmRZrqKrzKyurvNG78aFKdM6M9FyGZSr9LpqVmG35P798bq6nV1xYeClX2XM2nvWivMfWyYTEhzlUBjKtty8IEqnlCqLcocYkAlZFDnMhL8YbpvfxGrJtFyJP3F/eiBKp8y5ytORRLoj76O7roZ4w7o9q+aruGiE2PXqqjJJnoamzMKVCZZ1sLpOsiAoloMe3O/mzxBhWKooX9HzNOFZsiuRjTSl7Con1lThTBcWqDAsmMj+GCcLyTphju1YDCZ3mnE4aYF/lOLSUlzlLJgRE9ysmDq/zrMwDE3c/K9ZMWFiqmSVmKxI4jqqo2qVDILo9nFX7RXaL4x1iybVrBnvSviw/rW+rxobLBuWzWUk88n7lpO2zoQqs+DyHAgu4SJclAQbdPiNUY4scLls5/B7umhVc+STDimmPt2u920ETqbjFQIdj7gSMl0NyXkyJc9D6AMNDmFLYhPJNErBFlxwOtCA+vCLU+7BYQHtHH/hagNxBdRLCKSihTPFl2PEPU6YYwk1WC0yuDzjIyaQ+jjxDpGkdrEy5rfCekVw4gPLC3lvhfVCGPfm/LqxR4xgkmGHJHEqsgPh7OCn4g1NqP9D7m50K8L6nSIMcmWNB1SJtgdEPCyP1r9zju9c4O2xMJVDyLPsU5cgaNgB3TVJQjateKoHuDVejUACmkxCjU1T9uBrnGW4KcgB9LHShLlcQhPhDmWMHzbGRI4UGHsWHVCg7DlcP8MkNPO8sE4M+xA726Os6M7olCMZMcEW7L50GIHgaGE+F+abx94l2C1Al9A9XAZidynscYF2aBV0c2u7cac94nSY5wcYiXAdXjboC3wRnABYnALBkFMQGZ5raIG9YmgZVB4ztEG7uR00iNiKmmcq5vYfLmEQpLbFMmQzCV9CzCLEbom+OtcZLlMOrUM2J/px8BH8wcl0w+aCumWMdThFNVTyCyOcokiykr0MxNF+pl9K1hHETnlK0uGSBMYZox9O1aGzOIicdcgNUR8HMQR22QQh0eYa8u5wIB0OOqulxwu/BewiQCyyWBm/9WB4sGgfoyaESJ9ixx1OzYutpwK9yaEfWie+2Hkx4C7y/AZABETeICwanHGiK7K3dqvFMUeGj2XY9w03ToZvwAv6AZNxkl+IReQwWQ18HJiAT9M0Qcb96/vEfP1AaGQGak/lE7iAWF4OJ2mBhFi2fYdljNtY4N713cuwezn2CaQFXg58lrNZdexYQg3ptR1i4ECCs7YPAuSZqGJkOiHuA2/hsPSI2jd0zyXdSRQ6IuhNkumGOSFoM5d2GPo0/d1cnl2IR0HnJjguQBKYJKZ5OG8xTQVuF6L9uqy84abTNeaAVAYS1+GXEWqC2rJ5FchowF3zE8xsCtrQDphqyCMTOG1c6CMaIwRoPGxK5Ikx0QssR4FSFrIEKxwTztiLbqExnimT8ayQYwY8KxhGkUwGFDQmQIa3pN3c//CZqF9sPvm7TlHIkG7G+F7Mtcj4Xkxti2Qth6BlU7xC5+DClgISwfoSfYyhgucxZLM5rC4fFWugVY3PsihXhjgmk/Qs7ur3EqtkApWHqoXADzhCAycJaVWMb+XSjFxWkkSXemJuA4dsJXGp2t65vC3LgwnPmqNcTZQCY4wpVj6Mo70wVIuucDQV0EKSCU5IIGKEKA/hiLGdOBHKok3ZhcU4EYkThzjRCOlIjEfkUIbwCIMSji0oi7FEEYVuTsaZIcHq0oZkihEygohJQ8Qkxg1C78t4VOZCbOC8jW8j03qAy2N1cehXzqfCYUwB2UemICGTyeIv8UJlrBIZ4zZxFlGA0yiU9pIYRmHOh1ekFfGoASMZyewgzOSa8XFigXeXj/RqrKEQEo8euUImDkeu0EAImCjMlS2y2K3N9V+BH3RiJbAZftBQEOzL8ITeNF3cbrb3TwJXaPztskwm6EJJHrp0IV/drhegDmWnUYfa2loCeSjSk4eKueJcLUZG4A/F4ZfnD7WVfhZ/qZd3bQ2cn6rd+03LBgmTr7LIJXN8GQ6Q/dRXZjedNAKvXF1LTR06nwSi5j+dSAP5+6ZaLuuHI3LsL21wvC3BVH4VpQuXIPJKlaSitozgHOFIybhqF0QjD1bNEP31j+Vttbmpj6dW5Xp6Wz20sm71/nHX8awaMyBpB0O91V6DQ3PKUne5NmdV+RmO1uvVZVRfXddVUpoyTzOTBMtrUwZlaa6vqiK5Sq6uoyTuk0k3fa5uqlVVzyaTTiIdJeF5aCIorMML9NgA23fV7tf6oEHVDiSGjirjQkyG36cwJpkfYwIuDTAYrXCsoYCWoegzIDQGVwNEdyC8idEqcLmBR7T3zbiwE2huPz2SH5nYT+8fY2OR+jBTs96quLoO62VkgiK9NslVGZiiLmNzldZXQRFclcu6X28fH/fz1QYSLxyyMW5xqT2PhoA1nPQp6m+r3Xp7uOih2s0+6hbiZRjKAKbu50n8UpRMZilHgAQFh8hxwd1oCN4Bt7YDAqAIKRzBIOgHHi6ObkCQi7vcYtZBeJa/17HVHwcswp+fQE/sJTm4Tps1WV6F6XUaX5u4XNUmqZJmTV4XmYmTpL6u4qwKr/uyMH9dXlWHajO/LpuDjoZzmD5PAk7KO/3KjKLF346daw9ArSXfqtUjxUUo2mI1nF6t3sUKwVvAHUA8H6LoEFIe3LWQaYtcqlh2njzu4LYb8hHS9xImHG8j9yfm3zqvwfIe2uuuq0RULcM4LqJlo2cb3ZtcZytTVsvSxFfZVZUGqzpb9jUcPjVq91FRvyH0Fx4KX7Bqw8kshF7mJfJymhd5E+oUYvCQuLnTX6MWo98RiAJgBoI4ALyPfpIUJd2XnuN2ZHxkYT9SfBB7pLVoBjdxS4Gq41Ua1akpl1eJSZJGTpWruDLJsrqqg+vrqtEuPQV/u1utH+/nl06krNjzvJVzFvjbw9DTgb/T6ZUD05LTNML6oGxakCkJHNMM3oefKYOUiOYurOf47/iLA+bNBTAvBDbAjISSBxA1caG2YOoBtgns05AC3OfUwAC0GWLW4FnUcQixQGcJTOxAiFyYLAwumQAWvNcF8gL61savsRxRlNR5SIidW68HmgxnAEx9C7/qRaiC1xti2BYo9o/LcIhmkJIcX4awugD3hTsSNqaPNpFtB8+ZLOxBMBfE0CAKCqAykMybQniQrxuahhSZgdoXgA+EA4hc+wKqY2DNDNhmoR1jpVHDBUBmJATj8MADnUfroqDXm0wGThBSpOeSeYwIWATh0nRQSj9Yfgm9I2BpHPwOYAfR62/hgSHuAq+bwcQnNG0ZjQGk7KX1nMNr0bkWZg26DmMLl6WwHPrpGScJ0L8uKNg9q9IbpPQGOb0BvCMJoYRmjrZBv4cKeDvnWA8oIITy0bEHzeGMjZqNLHJNDLoLiGy8FB4JUCJMbTGiimgjki2NcCE+d/BykNjRLUECmF+oJYGwIAJkpDQIsH4dYBECiCyM0bgbU96aGsNAw84ZAp4QZXcMPYD+SFAfJnlsiDLg6ZCCYSfqRtCRJU+GkSWhBHAirEJAOxgQIDQVgtMIAT4AGQJrdYRMUx2izwARgqMSiV03OzVMHrwB5Is2cNzCgxUBhkbcM9cE/a2h21N5nWX4DUB7XM66beoDkogkhpygWkpLjUijEIQQSi8ArKCCJG1cwqT2Q0IZJNmmhPXrYGhCkLwkMME146KBSBwTeAd3ZgS9GCadfu67CoYQ7DQ8OMHJH6wUxD+QDYvgPspF7KJcGC5HTBAMDfTpjMFlwQgMaCbBFgWMHxarw4pI4N1C5eEk3AV/BCnfhL0NJCwF5BOWYKEdSL9jGn6Qy/1EwMruXhqUXfsJBh7w4WM3XRALQldIYwAuBTpiJdJF8T+FR4GTFyBI4B2Gi+nlaDjwqCAAaKzkurhhUfsigKVvArxJwITAaRQJANACjq4AD0GCgIwO8WBJAAuCUBA7f2zPOj4CQfLmYUseEpsAgoTlQvrnvy0Q5JiR9b8XHCSM4wk8SFrmi/3t9vFu9f6h3hxhG08f1ss2/1yX4IyjN5q3Xt9s6tV/tN+2Xonjz10EufMz9cmaOvjFom+/G5ZK/zeEhIc7vt8cmc15vFhVhyq87G7zZndY7w/N2qPnNBO0qZeHizFwfvIzwjjpH/K2WSTbDb/9XTO6F19Xu6tqt642zQK5fzjjEUXQP6LZ9u2HqIeEnfugHubw2o/p/Ya6x/yyuLnbXlV3x8VA4fs0Wqw3q2YZrx6d3/7sMRzBkGo2IMxAG/aexAYVmrzCAvjn8z8pM/CXytr7JTIQT2FvXi/n8Plgi+wrDqpos4u8EvLl+BYvhHz562/17uu63nyza5RpC+IJdSmG8V2TUpOrdYGIEm/i1t5UaPO2JqsgTpaxCeI4M0kVJKaqs5VJ8ioL4qCu6mWfiqaXKRiOGAwFFtJXJGTjIa2ifJn8bJ/q+0ZUDGASTY7LHlnSZXAbK85rkCbfNYJ3u7uMAjkzV/fzVHCjCP2pTEYn0xPz9zH2GeK7+1JiUNwIuW1Yk9O45CWsy4f0ouErl8fXHz3twIj11MzqjfUumdtB8FYIjxs70z9cWcw89AdcrBAFnsHwZAEHBDTt7YgHZDzBOAncGSMsFHfB8Aw+G4I0IaPY0IEB4hZOAAjCSXLcKATyTEieQU/IhLyN8CWGWsjNJ+dXgXrkITgKeXaVEDh8PNMK5FRhyVmwu6k1JqyC+nAcAtrh8fv5fCwIX1hgZEaoh26lPJiKD2FcyYne+HKnHE/tcJLE/epGeayOCaXPoTvot3Rrq7O8JhBrwYgOBmzIewaxG4i1WYEZodw5z97i4ueoKcV7QvKkzVZIZ4mvKYgCIZzZlCpOAAaTO4hZWjyBHeDLYNDRTY+C0RWkzlH8BQIkci4VO9ASIjWFegbCyQ7x8Aws8EA7iuGJociBE8xlgi3gwoJ3eth+9CxGjKHxcMIYyLjxQCMjagKPHIabxonxgRiTrO8WjIQbR6BX5klN4JGwFiG0x5iR/RuMEzsdIJDTpEAeFStxuwOgKqzZAdHU9x9TnkBCE7wQXNknlFBH//bQsaNrD6AOBgaX08D6P1haE0yF4qurzjOcdA+kOLnVCILibr11bzqUkl7CTqJPr4FJUErqBMuHUsJzIXUE1VpX50jRZj8JIREJFGZHTyfLdDKRHgXzqISQhKSkF5lKjhLSbJTUJ28Rd4qq8ZdzLqVOAH6EDFWMhwwvT8iMiQQsdt/ZpID71pofmlF8ArspwG0KuA+fPtw/fJohX4qDNDHETv7Dk6jFwpTQghWyrMyDTvDcAnemYAhiUezuu6XhWcl4xHF4oCAsAcspGBG8jjFZ43EA6cvIXnQcpDGXwyUGCcASutBn2GZIip0FfETDCh8XejRsM5CQnBzL94d7pKNvMdWLp777uTgPljnGwXcAkAMywOAuwRYwooimg1lNQB4nMMH+zDJjw/5si0fzYXIAQTACPoYdYB+dHbgHr3kCzSGFDF04HspsAUADiIkcMNsImzRAHvS3m84rw4AedpB2WAPjqY3aHp9MKwQGyUKKWKoDwEOgdTBuGTL4SEpvmMEcZyAF5+EfNCkkNPCsiz6bFNUCdNOGpuA1dFf0vUBfMhAQvHw4s1lBnDENi5iskE4QWNHdhbTgTOHqnkygAw26LrFtgcliMA0MegD6jeckxkH/Eqjq4XjKfhMANFj2PUVxm8CgsMw4KQxpQqOFyXUACwOR9BL64eBp7IQ7bD/0reHXYS2OY2UV7IZTA+Y0wUq/eKANCbwZ0ulcSs+DGX0kVE2/m2mdo/ZDEUxiF5xckGoEtgBsDBU4qH8cwwlhsp7ju9FCYuuSzmUhm2+AFA1YARgPhBJo4UGsyjtLGZTDqipgt2Sw6hE/ZKfqYf5ePxxIQCJBAXUZiYTZfHJ75XQ79ITUQ6xIe+qMOZPegHZy8UoinIlGOseRzmCkMxjpfBzpJ+YysFya/vw93ZmOrTc4/ga09Fiv3JRD5AxEJJSVamgSQUVOE8BdgRK2QFVu6XjMkkS3OAVrldNEeRMSZQsOUgcAtVuQneUrwnxH3bCz7cIMJYAF4oJCfBkvTQ+TxnYqCEbmHhoOTQGJzIxm3Mk4DTIOBtsBjWHSJPgavrVwZzaYjOdL6nqHig7oMHyiYAKZFUHT45Rll8FqkFYJEyyxXE3YHFM6FTSp4IUFjgHEKij2MoSkOJ4N9ocNihsGl/YNSRIyhV34G2LufFmaigVSJXAaeEnzAN5+OLr0DdAiCtlhEo+e+DbDOyPNF7N5TaHzAIfnAepB2ifwN9poOhdBN5W/KRyPosPw0F1pjaLoY3tdyuMUjmdZejT4O2GHu5XiEXtHh+iJVFCGWqDqc9M/5eN6KBZcIeAxie8S0EPoOcIB4Mi/oTkPgIEflUnNmJCCcRmNSEEEokzBBbOF9M+/4IIvARf89nHzsP19Ve8EuOD4W1tGdwIumGXBF4EL4nqZB9s1ahlBgx/bpSXhBruZuuhAMec+LCHwYIsDedz7cHDB68Dg8iLVpMj6/yEO7pUwau0urlr58anetXnSdp2S+Bdy7eWRa8Wr1S9ffLxt8y79dVPft1r1xRByvZOlLb17lYVXcVCYIm8Lm5fptSniIDHxdZZfpVWcrurrDiCH8gZRckxJshK8CqgcK8ZbPq8uYI+M68BpQo3zHg7nAuEcjJ0XUXeE0imqoSeLv2/Wv7V76tDYOqkIoqMWU0C6OPID6QADAugCKyKN1h/aVHD4xibct8lSeJf8I5xtwNPc34ufW4UaYlYv3dfATuKpydMx1oVnwecAKYCAM4K0QeofODmRR5HOx3RMc3MVYCEwAMPxgihgHAsZEwxch10F9MgcTM7KhICoF+gy8qvAr0NeRAJA0bkfbW0giwFrDtyq4MefgtkB7k1E7xnE4RH+DWFxBr/H1Q/gK5ZLgG7mz1oAYAzAYnjgZxDDYFA0fuPp7Av0aJaIfujl2AUn/cJEBbNx1Q2rC9xjrCAZouJ4PTI34wEULDOAoAO5g99ixTLMigCOcKxjRjeR0x94sHe8WBkgvOhMiTg52OTk+nEhcSDGEHyH6DesHIbVyVj9M4CmIRQugVvDHoQtTxJoEtaHIDvMUMBkggCFA9zcbEUyANaxGma8nhl1BN0I4K9x0H0wV6y0GULdLFQeOooQxwe6BrObE5THg9IDrA9CgCAZAkD12ObBGXTgex5Unl2QDOdCwOa5WQuYVsdhArwfL4qGvbQhfT7snxfYx0qbWSBAvmQd9B8mMJeReiKoDxShDAXkqELskXXnEVyIiwy86LR2XIigDCJk9dEQ9se/HWU3/cpCJfSZie3RWUqDjSEDQOLhdIgYQvtaBw7oAABxQMHVPYP5K62bTyH7oCNWgTP4wWoOgBx2bwfOZ+ahfT7EnrfoGTTyYPdkuJ5V7oz314Xoybg86KzBz3QPgyg+6J+D1YPxpDcYLFeAnpX84YCFQWiAA7gDkF0EMzl+Hr+S0Hg0xxHON8ORAZAMS50hxM4ab3AxD3eEMDBmhPLg+wANCPeG/FILjs10IX2+4msiAA+welaltdB+hQJHCBYT7B8cN8QduoBBiMTQmMQwcbOoPjbbgNi04X1/IHzPYKQkhR6PZwJY+wybhnFFb/k3AtMhTMnB9tEnjLLA59J6D8DQwIrB62D1IOoaQGIOONBXJg42IABGfPhBQgCyvUi7A+BwhOhEeBbCCgETBDvXwgciosst8oboP8DzIRLQBgyymnJYcs7NFMWgg+Ptadsg1A9LzBnAA8JHPOIuVOXmCAlIGwvOqBb4z0XyMToXtMayc1iZDhALgCEUC9qxKnbwdEhNBdA+kCkcpOeCBA0iAgF2aE26C+4bbjxduw7Qi3hzAEMBIJ4Ogz4wIGD2AApIQhCOlh4AINzBcVqNO5bOuA4CEKthuXi98VsBCoiYQThEQ4QbgLKAJ6IVh0QXPEuCZYPnJYBnTGbI4nVLmVAGsAW9xACxncxOhTA6CYkH4D/A+8HxGlYkgJAA+DwJx5MxeChzh07P5Ooif8pgxgD6jvB5AL2D3wEqN5Ojy8AN0A9CT2BV9vAcHrpAORdPx2B4gHEbQXvoLxpoAnD+R8ECYAcXmCej8Rgwjm9nPgIolFwg3UglwQXLpCmA+EAAgSsMgagA0nHhZqM/YHwEehD4kJAAltFsPEMY+ggZ8COhGYAlDxBMzEkHT5pCuiEqjjoI+DcfuI3gb8Zu7WDWoC1i4hD6xqv7ASYJbH8EowFGrbDGBocMPNQIk4O5Ice1DUZDryoA06iBjHdDrFppjZ0DPQM8GkDWAD4G5wkDjQ0kUJNxaugeK52xdfJU03ZHthFJPkRlIZYLIWESZAy9Y5DJDBKZMdAbuKVExNvYNxKFyMygNU1GACKdWVgNEI5kgTwvARxAzLDeIHVjPnWbDCLjZQLpe0zuhgUDDcG1CH/GAWEk9CDyApkNML0jOF9RxjiQslkkGtQcRMAYwL8M4b8ALQZfBtY+GRYpw59ZOLNBOkHQyqk6mJYFJZs7IlGmwGPRQvrnX+CxlwCPvauWt82Q7KRcc+Nvl1EWTYDH8ih7IfDYRA644yqZR3FFgT8F3EwmO+UT0oiB0oYl5UOLNdvjLLhYPAMXCyWwWHLEl+x/qK8P32x33To+jL+3wufZWLL/U3Kf2cCh/yMyof0XwnxlyRfEfFV3d0/Oy75Yqb3siANbBXkULavcFGUWmSS4XpoyDGtTBVWcBdfJMl5WfaK0oyRhedI6HceqtiiwXxmrNfW88i1v17vl496qtNcIiRsB9tU8cdMpnP25ELBmO91vJ6vEAP4rTJNnAMCKeLJgnw0x4iYxhPnpCEoHHDpaEJSHbuMH9YjgDYDeUBShi/jINYoSq8O51OHc6tyzoF2IjIJDfCwMjQuKQoTVHKzK8AlBtA+DeQEgTJpMGYpGj/GAswBcdRY4C2BYsygrgA54MqOBS5qlJAMYFqKgRrwAO+8CzIVmaxIiJacwAxwXywMGAC0IhPvQUwCZAoglJh6DSaPj6BRCDJBo4Fr2pCITk4PJ9WTAZ4moLABrIe6NA6oE6JSdAowmw0EfASZJBiKBT41nK6OvAAFB6xE8Y0jYGlbL+ETykNi4JoAR2RAoF3MEQCOEL7mQJWPflgF0fCAqhEoh8CCyRochiiy474BAG9eUi2BK+LDDxeB9OQeV5KCPGMoIvBjo1O5fi5YMvrSDK5LxR5ajA0E8+HSOD2KAHze92Dx6SAb2yFnAhHReANrxpwgTEoOBzLARPC7GR4HZQRCOB7/DQDsA5pExO4jCmcfvnAvTOX6LkBVgzzIkzyR6BxE78yAdROBYSbWgg9NoHDKxfLgcGYBjf3sy0ubJxdP84UlrJefDgstGhM2TBLpBOA0ibhZPCH/BfLsW4GacTcDdeLJqjeMT0RqA4fFhas6shgarGcEvCF/xQVnogSPwhf0IA0uacjrR1R/n5LSKYa8g1gXQKgKc5WkGAEN4hpI/G3pkAMbgQl1GIAt9Q5FVjlCBO4x4k6n8U3yJQyIW6qeDU0FQC1A1EN8CrROaK4ClADoBQSxgqSPgZAqa8oRhejJ1wz7MIWFVAJTCcCswkPA1tH4GbgVBIgBFQcoFQVEgl9UQT2SqQiiP5sO3+PAkAEmZBZRAOSeUjRa4ZLjQeoiT54oegfNJN8xAWoq4lHkkiWF5qehWagRLRpJgOCVgUGcIyDt5pBCRkliDzVbfVGapFAbAQaOk1DMAlQCmhEFQ4HvEoEziTvzIlv6tGSAEQCAIqURACGJGRHAIwEBknIg/B9QUeAVgMjDhfWUocquEoHow/GkjaDyYEw2CBCEhlOwMItIOPAWAJoj3eFkgCWSIQrTKuaCSHNctLS5YZwLMBJAlMkoEsScMkwJHPwBgAGQEiVkjWoPkVsGHFc/ItGvpyEv3pN0H4A0nYRNBSHhBOQjR5vRIBxEC/ZCBJCFalYAeAHAIwkdkzAjkVAJwCCBJyHJiqY88+A5AhnjyG0m4D7iBlaRoOs8R3QEhIADaIHgGADwQnOEDe2A2I8B4EAbEugclUgFV0jMMacmju4tWGK47UuvtVaP8Aasa2YjAO/KnELKzDdGMoMR1gBXQHRmlgcgLQH2QOwITFCH+A2x+D9wFKUyDnYVOCMS1y+AN+ghoCiv7ziSYwsrlAyRNrGHJlroI5PAjQlhJPoBYWBgMA2+Eq9NO3YPJevq2I9CiDEoCWrDw7L+K+/1zABdvb9e/VXci3mL46TIusgm4RdHMxqvDLdhaUVTeG8r7fXy8qXafEXXx4/bx8Iw7Z6UXWXFmIp4oncvEEyT/nSrSOUiGf2En5t/gL/Xyrn2Hn6rd+83fW4hD8lWcc0RFs1D+2YgKa6Shd1H+T8ypo8ig0x9i1qvLKAqypLquTREsr0wSB1emWJaJyZKrNMjSbBnkRQ+cYLKBASi47kIgRXR6vbnkWUiK5kOjf456i2Eq+rQ6r5BP5yQwRV48J5mOH0tBDh04ctM5nLsLyVtDbmiRnQH5v+1TAjhL4BRBvpKEmVZwI+B+SJl1+DnEpmZZLprCfn5i9Uo+yuILPg+QIWEdCIjhoC0AwOEBXhCMgiE8qAFE+CDLDVSjA+Ij8xGMFijEp0toR4NCcAiKsNtwDIRbALICsjIAwhiJjeBvZPANGWYBGW4oVgiAijE0OZrsFIq0S7gNhzMIdIKLRQRgOKALKOs2Fnt74uAKfAbEN8m5KqOPuM8MPAk2tInVegNoAyAwCt5fGEoXrjCLbbABCXImmdnSaYR8gCULEBvyVMHh1gEzyBlfWA02cIFhqTRIPpvSeICrGa4G8EpEV0Nn+wZjM9ybU9gFQCbQ6yCQI6bHwSsQPwoZNNDh4dXpK+j3cH96DPpEaQAxmg7sMBhhiEoPPSXZBsABghNAThEkJtG1gHSAogtChg+nkBh+PZ/fw4EBOOiAaUgAwD54Y4iAhdDgHEiAJ0MHogASe0CccL8c7XdAAGfE+OW4Pov8Q1QDS1qdFPu3Av0Y9WXpNCD070b2IVUFj8ZjeSIn2i8G5t0Av50ew61idWZIHtNXTITk5ai7L04PIfl/bhTe8JFAYMD5QXY5wYThAf2pKPtEjgiMvEPc3BNaT6xxhk1j14gii1uIkUOoPYEBxxaeMLk1FE6QfAyE83UKe9fNsWAVZSI/eF/vyc34wGLc8D3ElD2RbynGjVFpMWidWu+Hser5kDiEtCHebOdXGGtZ0ld26FsOUfui4BCBlpMdYCwaGlOD8XchNm1Hi1msGl6HBxXcTAqp9SJCvBojyGBUjoNGIFIafx62xnOeHCKGUDApWDceDEFp6JUnbA0oHQh8w/MhpoyZBWh5IEoYlw2sSjjpzoeRYfS7acUgK1DxQarDxFuh5T8w+gvh4ZyGcjY6jKsEJNxZEWExpipHTyHSKpL6WTAOY014xDs3ZMyq3riBYwr52bkAIM6IkSMnautJFoBhW7JM5oO2EKCV46/wZWoNJs2pG6rlRHonnjrL1UdSPoZv4bBCgVBFVJUklRyupWMmRDxB6iFPn2KxLI45PMyKdRrqEHhoQF6AFIEDNSxAQIRFNAqg/cBf0lk5WKtPTYsXmfAQIiVzb5bobkVe7QApMNeBXM9isBhCPYW6nvOXQE56noyh0iED/79KmvxXLmkShlOs9KyZ4BcKk9413br4utpdVbt1tWlG9v6BAprDWpkPZRYDN71Zye2H6LJLRXr2k/o4mO45nihqnL1SOZM8eKVyJq8e3lQHNH23SIXwYvyVFV3MgtdihkfnV//QVn55ncDj5y4SJ5TjoAJo69XlMls1azyMTVkmmUmqemWuVnVoqvAquSrSMrxKsy6gOOwYDCWOsv1UNjavxBE9K4Zox//6EGLHx/YGA/t4YhcTdKOKk2U32gBiIxpSMTjYxxcnAoNhEPrLbPDydCFSRzrIHa/txqDYkFhpvIbhxxh836oOFzh34H3Agnkl+wVRzvxJYjEOf/dC5/3EBxV2Twupo+PrPS9miAkqWS8wyzpmMOI1M5DkwTMtJ6yzDPLcJ5zGcWX3K7E7jMnBnsSzlPNkkqwcKns95AVDnJDV/0Nanzx7fKUxMH3fnYh1hxE8+U3C2ZmO2Q0TdkMkHDPqNxsv5G54y1KwK9g+QKg3Twed4jV0CRsrxBd39+UzxzpJTn9r8bC7sd3KFiaba6zAmuC7gOucvwo7gPHlyg6CIQ5Gjm8DFKWxOjwr3T76xtEbxIpWRIEzQz0in4HaFVmZ2WrG1F8R7gWMY7IKE/6aFH0f2X5kA5ljDwp8aIq9tvlo1DjBsWI7gMkc9LCwyhDd9Tlez2hihoka5pZF2RDiEotx1EgEOFI5xtFLsHNshtnL9Thw5pwQSGEs/TDkHB67muHrRzBKOHr4ZMBNjO+WuQs1wtcY+oK6go01jiuLR8f4NLYUmMAclzZei8sV34ZNcoAPyOAKXv+WM9n75jkOcYyjmuAjWKdwBSMAAOkwbNfh+ixw/FBlDe/aGROYcwJZ30MNd1aIvtcJ7JcCb53iQLMu44uV2DWmbZDdQZOSsCGGd89xtNA7j9kzkO9ijQR7K0oriO0Rf8MTD7I7sdFmT8d6v4gxwL6DvwlztWNaFG7TcKUfMvnK9Mc4IxFbtGgYsZdHhVrgk0pXqluJXBkZGDvBO8TGnHWB7XjUwnzFMp3MtyEuMqBRsDHFy5kBP/YMzSWunwN3yzBKT4avg8uIHQ7wMxuNAjsAznO2VtDIBBbdsIZQKQqHHvFwxIpzDzYhUzTjxGPxDOcw4TkeMds/5PYFU3ss4ynLbcoOPnZ+UpaNHVNUs33FDG9++Bt7w00XUp40DjgMOOJMnY53D9jd2SGBFbrGYtj+tKaUv/SXxWp9fX10uz5dftitN82hvT0Iv6seLv+8rx4+VLvqvj4cnRuHp4f68i/1ddU0Xuxvq+avH+vlodrcPN5Vu8W+bo9vWRAlZRBGZdDm72tv9cN6f2iOcQ/b/Xpwf0R9tPiq2tdtsdpqvbl8v6yrNr0la9fjnWbb9e82205zr0HqzDYc9PVcw3CwjGYbDlp8tuGwomcbDht6tuGgSWYbDibBbMPBxJptONqosy1HWTPbctybcy1HJTjXcNQfcw1H+2Wu4Wi/zjUc7b65hsmgVeYadoJBMTmdVa2Zm2PD2alpGylmpUcpKKalD5Aq5qVHhysmJupRR067xaH765u6OoZkvl/WksTSSLZYKdkipWRTCyzN7im1kq3QSrZcK9kyrWRLtZIt0Uq2WCvZIq1kC9WSLdBKtlIr2AqtYMu1gi3TCrZUK9gSrWCLtYIt0gq2UCfYVEJtlFcayRapJVuslWxR4pFXktDRCKdIKZxCpXDS7IBSK5wKrXDKtcIp0wqnVCucEq1wirXCKdIKp1ArnAKtcCq1sqnQyqZcK5syrWxKtbIp0cqmWCubIq1sCnWySWlwGaXFNYocjXCK1cIpOUk4SROjMKZipVyLlHItVMq1QCfXVHqm0Mq1XCvXMq1cS7VyLdHKtVgr1yKtXAu1ci1QyrVSK9YKrVjLtWIt04q1VCvWEq1Yi7ViLdKKtVAn1pTmllHaW6O00si1WC3XEq1ci2KPtJJEjkY0hUrRFOhEU1gqRZNmS+Va0ZRpRVOqFU2JVjTFWtEUaUVTqBVNgVI0lUrJVGglU66VTJlWMqVayZRoJVOslUyRVjKFOsmkNLaM0toaBY5GNMVq0ZSoRVPqFU1KA0kaWsWlkVLQhUpBF+gEXVgqBV2hFHSanZxpBV2qFXSJVtDFWkEXaQVdqBV0gVLQlUo5VyjlXK6Vc5lWzqVaOZdo5VyslXORVs6FOjmntL6M0vwaxJdK0MVqQZeoBV2qFXRR5JFVksDRCKZAJ5jCUimYCqVgypWCSR1109ws0QqmWCuYIq1gCrWCKVAKplIplwqlXMqVcinTyqVUK5cSrVyKtXIp0sqlUCeXlLaXURpfo7jRCCZ1kLEXNxrBlKoFU3aSYNL4+UKlAAt0AiwslQKsUAqwXCnAMqUAUzu6rZu93Vb7gyiXIkXDSBBgx4aLXZ/V8/Kb9f5WlFKa0QwEcSZ1pBSkmdWP9+s7SWTls51frO8fdtvf6pY6c/xivbm5+HpbHfaSaCvmO5sJIlBql2pFYKIVgbFWBEZaERjqRKDSLDNKu2yUbBoZqA5H9mhPjQxM1TIw08rAKFRKtkAn2cJSKdkKpWTLlZItU0q2VCnZkpPFSyzIue/Wd3f2QfubbXMbWaZFwtXjM9/cPd6vN4/3oliTpKEo1qT3ksRa6nZFEmTWkvhLva93zThtq1XPhPyx+dhcuGyJ0Otq8/fN+nD55/b3ltvbkz6PrMWW91fvFo3Iu9+2D+johY6cy6cGqHmXOyY23603rU/jvnnyodo9+Z799rbarbeHi451vlh2JaTfDXdJWK/+nw+NhBxIvh/oh9DqbiaIZd7dv++qzfrxXhgsRxwrxHaiFduxVmxHWrEd6sS20mo1SrN1lMYaua2O1Bo1Pq6H7GvkdqaW2/lJclsDdgl08j0slfK9UMr3XCnfM6V8T135Lgq5xJXvney2pPCbw9aRaPYa/nDX/Nu2t0bew3zL0CPTRcEsPpukxfebZb3Z10y+NW02hy5jxm21/3p7OGzvf2xTHvfZHlypnggCyZaYeK+b24P3hoWgJrzy35HkouKQTNZcfgIJ/aanG8+wSLK0UDw4UQrdWCt0I63QDXVCV2knG6WhPMpSjdRVx5GNGrvXy1KN1M3UUjfXSt0o0MnSsFTK0kIpS3OlLM38snTOtvHYNJFr08AXPx+zK9zUF9//ojF1QoGq4ZXN1tL+uNn+LhnSKvkaqYWwYJaLQnheWEtEjV5WsxN7tbuXhDJ990N9zWXrS5upOZvSN8vmgmr5pJxTiWbCzdePt3X9oJH0qWKWJA6KKJklDopfhGtkvcRBkVavnoKik/VK49qI1rXUQSM6hucPrkaM04tPUCMjjZqP0gt0jejP1aK/OEn0ayB6pVJFFEoVkStVRKZUEQIdReGyEJgoPoEsylqteSqZ0DoLWtNI4qyILSXOithQ4qy8a1OENR+85vGJ4t6Rk5K0FV1HbJB/OMryi/tOU0k6ZMx/ddQiHx/qRjVVG8EkqI7pkdqHHNbXTxeOjZAcEz65fhz+gG/bLD0X39bNL9XdjOEh+UzmHU8SrcevAzTKQqL1iA0lWo9o6uiUhfJQYMRTwWykwYj+9G931X7f7MwVXXrMfGjvASPiH7gN0KZglHWERpuIHCCxZSZpE7FlrtYmhVabhKVSRxRKHZErdUSm1BGpUkcIFKATXCOuqvj0uFntKr17O1IuRK5Emg42t5Ws+nD2fm+rw+Gu1twvmJU6EvnIrzxm1OWxZOD5vhyJtiT6ciQzO5HknD2L76v9eu/x23N533vuverkzeOhzYFYTwl+iTRlO3paWYPvB/lRj16rNrfh/tP2exiBWNIHGsUhUa7s/vx02wyVpB4UsVgdAUt53jDigWPihDB/yDUiGEWzwoyIDuZa49v6fj97H5Gt1RunkpUzJpAd7Jzl9vFwqpFjLUsj0u/bTrDuU+7aerOs7h/aL2W9pNFghVqDlSdpMHEpzYOAw8Kj/RSX5kqFmCkVYqpUiAKdzAs4SSSdJmkvrxn1Ejb6yS472wUn0NZAH046aUKBynb+2zGVqbzp//24ubHU9Kdd1eaPvWhuZ0+bxJZTx9klBp17sjvZN3eem/VEn5xE6vtOe+471XvY7Hvs6tum4U7ZT4lTeNZp1tHdwtnw5Jt+GYtGYkHaFkSbInxOD0okydHqlGa9z0h9fJFPu/rqsZluSQ86zmKNo19l3mhMm9FqCdRWy7wL34hAs2F3y4aK4P0CK/2w3XCx9B+P1W73JNsqiWirCPaEbNQI5oJgeMjWgsauKLV2RVg8w1rIz7cWMqW1kCqthcS1FrxWgCRYeKRDMgkEGecRMALU6XT97rVA7MCTpODnD98acSRQS09764l3LE4yQSS9/oIOAYnO6nNFxtNivkMoSOpsBo2hOOFG0sF13oMqEWGlmKhX+UhL7gW1UXmm7eFRTzQXH+pqd7dXY6uN7qA+6iiNNhPxd2cckEViSXefSXPfiERfrvgGaCLrQ7V5vK6ONUqUR/Bms1UPbSLI3m76WB8u/v6waLZgW7nh0+1639VusWc/k4/hGuySrEcVCrfwKVzLGpC16jScv1t0ztIKJIUsaqihmNO5CvlcZZ4pHd+pUnMnSs0tUJY1QxoKfGVJqgl0ZR8gRD5t2RpxXk8ITOVJ9T133tTKUhtDPasgdaBkiSytNkckAvXohZ0/yL7b3la//3rxU2OIr7eKI/exYNarhi0lorffrSy736fHWyJ+i0+QiN8TLgJUzx/Wv9ZyXHjah3F/9bi/1WpnCb8/J8N1p85BB6tJziplLXKcTnc76FTyPDSOstjrDpCKI2mh1ZClpCHVfjdIez7zoDD3aTtJPakAWalSjyVKPRYr9ZhAZxf1u0Bn9yNd5jWXwGb3qCTfoVA4XPmkva0ZXuSYKXHe5TjlN3fb7eriwTmUwNnpCMWUzk5TQPcX0ENzcJpmNlGQfrzdLn/1yNEvo8Kks8Tpx9EvMHIR++bnv+yank5ooFcdOSmZgu0BcGENMJa9/0FzupfyMYhh6VmPkCLe/q45zd15vJ8veNQ/H3guWzovEo7Dd/kioR0juVqmAnhzvZ6A+U0FvLUABqNFTRuRl+gDzKvMpNA1IU532Ytp0aSYgpiHdsKkdnAIZ+It7ZkRawucBUUoJYtPRTWGMjiaCAvV4DnRtz5UUtdZgNqkXSpTMVGairHSVFQmDgmFxCFeDrdoKnpi6hPwUCG3iM+qE9KLTNmKSrNSSDIi2ItfV5vm/xYo0MvwU/gkFKZheBLIW2X0aNVIFEimmXQC1UHLpaQqtqr/uL5rrlaFojVuDinPitDQRw162aO15BE5F4gy9ywpWO93i83dTQKl+xSnvLzbu4rL2xOvQwPinCilkXizA/RVo9pFM0FW3Qo4gJiU2e8DmT8yU126uTtSzbozvSViShrJW5JpdWWq1ZWJVlfGSl0ZubrS6wZRIBlCIRXNOZJDUIe6pCc+pel5PDGwHnd8675tLDRBfc4TNtpMJ4IyVepdIQUOVyheZZIo9qeUBseGvzuHVI27RlJHKgawoLembCZbSYkvfbI0l47c6pCJkQL2wrr4+LBe2vwJ4hVo9I062u/KfVFViXK+nJe0YiJGfnToHBSy0BfOiLLQV2gHsfzbhNDXqAcxx43YUsxx80P1q8PX6jTBeUmiO+WgTWqs0iKxVotErhZR5vIrZocvFLLfSIto3mEgpMcZYcObdliru5+2m1UjNt/frS6+qdaH2+tHSXnIJu/MBhOS6ahlh5BgZ0oxzLtEhAw7cwb1K1BmzxXoA5Ff7et+2xzsVvWdRiVJ+X6mLBNb0ygsUOlsBf6XLwNuNlI8+iS0nUoXaU9tJ5l5RiIFqydp0GBCYjdRg3nOQhN2HBXynnuAWJzGr8AUmo5KCAstJRfHm111NaYnGdBe3qXV479sVvnz/OZihiMNDC4UMx6JmjV9hmZNtJo11mrWSNCsIo9X6E3oKtsJ/O67dbPdngFaFvIqyX5mDexYCR0RAyP0HlORqcllmp8dp+r0vhSqmk+e8wIOMsEc8JoSXrKtL0ivOKo6OkAZ+zw1On8yHUsLMY4Eh6yWaneqH9k+SGswduecnaf9wTb+WBH5DyVTSERw2EaPbHDOO9TPhECaCXfAVAKv59itr/cyEiLAI1perxMS6UDv2pBI+Uq3g5QMTO9jlTL0+k5GRiogoc6kIlb1GpnGomG1Ofxe7x7mkJm3dXV3uL3Ms3mwUSyiNhdT2Blb8BmxymufBWQcgr/UVihJ8HIOlmY0fauP67tffZqD3FoKeTeYiWICE49hqHHjiJnT/CbkefXiOxPyTPMz1pqfkdb8DH3mp+R7KcVBfAVbU8jPdhYwRMjf9gJGl5Dt7YVMOdlApM345HUHC9ac7Omeden4ggVT9uHZUM3ZYLfXNHt+uEGOOc6cOPzSe1Zm86Pxu3q1Xi7mYKJsC2rsr3MzKRhldmsj4QbOivT/s4yuI4nkWV4Kifgh597WVU54BZeelF/BlibScV2xlew0TEZK4PeshJJGyhSuX8lSCR2vJSiVOJRCcRMZvmQHnRTsmuXGBD4wxPylYlJBN7/f/I3ENIMK7mSoyDDYHkVFe+hMWyr22VKiPaQxnEKt4RQoDSch3aAHYSkkHLS3kTqjnpCUUHcKE7IUeojyipRMvhSFgp6frv/1021151jhQhJCL4DNpztnLzbKhONGAjDId5Q4Bk5GQeus5Eko+FLhNyMlMHgOrHs+LsoJ6K9K6jBSPnJtPi0zX6moy50+vymNVN9S71qRqo7bZ+z3a64qrMRbs86Jsx3q4XNKDgyKT/SpzEZ+xISJE+gLTehMzJgothQzJIpOg6OikzK7ahXdmUoy0irJUKskA6WSlBIVSgpASEooNRMSEPpKI4opHKyhkhLjCjkJfYpNkaNXyEjoKyejKbih0nd+XcfX7XTMQEqHarxQC5VZLkHXe70w730wEitfxiFq1JPCKDFS+nJ/kmqfdhFVgKBNRGGvzaF0WsbFXtrOaBKd41dM+zaVSVqL6wjVaeJCMU/chKxWlkg4V1RHPlEtiluNXA6UcrlUimUpI9w8SE/IBeeTvgohrUz7JmR98wlaz1aVTJ7/eKyv6uXF2/Xh6QuyQoUcc6/USWXyxxmes6sDdOctIxy4zoNpnnCeUik9iRevCPZPH52kpHF+haIQDEaqs+RPLq2puFAKWmdoKB292JlgPtdfr1E0TONQysGmOe0PGuEEXePRFIqIp++oBAC5IRBgn4NOymUgqSeNIvMmfRMVmSbJduTTeKLW0qi3QFBvU/Lu3f/4utr8ozL/vt3cbM/PnV0KytJ35hCz7GuOCVIathMwy0ImthcIIAp528SNpc6XqEzw5svvJpxyFCDjIb2mSiWemeXqlPTXQno5RZIvj3KVNKWM3tJQiDQ5lD29+JJhqHlS+DEwoTm3Kfa1kfKq+89tGn0slcb1l3DQ1NPr9KWmnl4o5UuTW05Vs9JqMy8oKVOAksRN9v1iDo0pGiDQN+75nTRDxFRwk8dRO1Wi58yo8S5q0751ulajlEOtUg4EpewLRsmwqckqYIWgUyd4BOdqcgeXfYqclvLMCYMvZZl7djUDISOdJ8Yn5KRTGx1CnjqfnhWj8YI7UUwbM6nWfIkg5jax5hDhHkn13kUhGKihjQiobM/R8WQTYEyDoNVZvjomnJD4YwsY2l5ffLu+ar5stp6o1jT6TyoK7FdrGv3XqTWVApQyZE2oNZUCVGexCsU0VqJz0Zukymkp5qASA0FH+a9NI6RSFIGgKNRypRSUh05ntnhESVVMgkfc1EVSgqjXUS/nHwOk3FQTeuMkN7qUzgr0h3IMPu3Wu+YR87WArEjtttodIwUfqt1B68oU8mqpEsMIebYc5IMf2SsCjd2BOAdwLCT20sCuhDxfinFw9K435Zr37GmtuqG+o6MSZRfvMwF/Ai7X9h3bOAQfVWQmDNl5A87MgPEsRvNpKZgchoKUUt1/CFXWOJLqWPvdvRrN3ilslWqXMm9NKGyVapdyYsktpVRU2rPtOV7EQfEHkiabwaOIGa2kc2CotQMCpR1Q+swASUlPAjV7KvszEuHNeguy5x315p3fUi4tz9aSsmnp8JJSei2N5hCybXn6JuTbctDVv213T3NWhpCPS3He05wkjXCU9MkwT6Gt0xKKGwEx6o/4eVTV/NHNdzCcEzg6S8wBKZ5+epXKZJ2J2pTKXCtRjnO3Ln1q6+zARK+3VBpOytQ1obdUGk7KrfUF8AangiqlPF7yediXx0up614D/yNm/joHpNSpWU1230Cpj0ulOi6ecyrPBR09z8KQsoYJqlFKEPYlTtynrWApO9mziEVSFrMTK+D9M3wPQk61STKnEgcoJF1zXGAeFolkAMyzRowQAvZDemYqI3jiNQJNVeqJkBvNaxcoXdunUxLkpT07p3AeZemMFXk8TsmUpZXUUgFvZcaHUjA63EttF4WN0zqhcsnpVpoWT9ebJJrTeShlfZNbSpnZ5JZSirUJ37dQuGlW10up0vQHYhluPOM91eTYHUyFyeyi7Xl21pkXKA2FUmknFEo7IVce2zOPSSAp93kbQEplZmtXWchKuc08ZWlVOcCFNGZCO88pWFJFspSbrFJhhLCpF8QvwI1t9eSruSFQImUVKOTjmoyGvpqWmvH0ziZ/VohrV0252lGXN9ubtljpJHwxm61ngZ/ZmzNjOUehUd1d/FitV/qedprLIwk03Dtl+i1VBsdzUj9KGas8iukllsHJSS2lnFiv2L+TlmkoZd3Sa3Vru3kDWVLCLfsw7y1TJCbc8iYpChVFDEqlpi+Umj5XavrsVI+A8gz6+tVee7Ni3vqQ0lvNF4mSEludmARaYZFIya5ES0PnmTfCCVo2SoTwstxQwFLzweszQJyYTfM5wGmNFBH4sH6rRhDxUvh0NnQ95vJWeMlnimEOxwHJiHLG7ixL6tSS4dyWOD2+NFDqTz1ca6yzcCIz+nlaf1peTJbBOL1WkmA8aLIDSimVvIJByolkl/c7q7tSaiRlSu/zwlKhlE5J9MPPzbSUaskLUlYo8UKpxHOfEp8n82ZKvZ4qT/BSriNhsUlpjUR9IaU18um9OfUox7EllTd/Vu9Vnhy21ih0I4CdFSWjrXxdE+mbp/IMZCc/d0IdO+pPdhyo3krMyiCkAPdUBFToSVVQXag1/mwI/fG8PBPrUefZ8cR4TsNLDMkNTvDBCJrthbSklDPKr6VODhv2CktTwUjKpTQH8O2ViKy2BCWhillL6ZB8Z71IKl00L/4LnyaavzT3KSel0lEcOlOlcpLyDvnSsHlw4JYk8RgbHBmmqCBy1+5n68ARJa9d413Kh+RTmvMV6eZD1KK7WTqKev3NQgBYJHvKAV2duBLIvgrVNCfITkmicLrancrwf2oGPCH98gu8v6RjNZljJZV7Qo2sU3PAOWfcFw6Olh6LYU7tBa5RoMyGelY8OpQyXMk5OGUNfZ4TfSK31WyPpXRWXB3rsr5K2a5eDsc1X/m21/wKfmOv+ZXe2zM1f36+5s+Ux9JUqeYTQc2fxBm1db9YJ0HKWuVTinO6cz6urFuTwqlViAL+tN5o80pLEenJIK/SDyywen1NhbxXKokm5OV9UfV8usIUClqd0SNJ0Z2jIMtTwuUTwuulteMLhWc9iS9es7dnEQO7Q7OG+GSLCHXdbV+iBk2latkUl9KCnXtQl7KB+c/R0kp/AXWr9VKr84eFUgIxERsfSZnAvMpZSlaoU85n6vXsfL2eKvV6otTrUmouXz4HWYVruMeS/pbzfM+fEn1O6Xkr1eOmnttNUlhXHzWT0N5ebS8wml+K3m2ExFzP3+lClmMtZeh0O8KP9nr2uVuoov0CZkTpMSNmplOHNw1cW8MP9FXkbe3OnSqlOV+h8wRzOJxOz3We7jtbj35pvLaUJ0wFfJLyhvlPtCLaZ5ytD3W1uzuF+KZOMRZpU4zl5yvi7HxFnCqDvIlS68ZKrSvl3vIlW7e1rq+Akq0kRf0q6sQXqKMl6sxJIqcGC28kjPdM/nFJpU7p33kn/8e6zc9wKtRGyMb13ekFxxxlWzxbSanDxEL1nqmcEM+IV0+/dunqZlktBa66nU8o3ulHRQL0Tj+qFKmQz8vDgnlZrXemv1nK+3Uimuc1zvDqFGKhlENs4gSp4WFH6iRikTaJ2FHFSetHqeLO046pTztKKk6hCmOlKlSnF1OpQlcNSoBUUbt595ioPiYBBFLM9nnGr4Q4PkfB+XTkS8XI5WpL0fmIfynbmA+DK6YEsFBFTtHLE/zH0qn4mWfJk2PYktJT+cQCVzv6U1CoclUIGbv8au4El/ALoECFZF2qCK4kO3TBSOtp09QSTWLoUMqfJY+vOoFWpE6gFWkTaGU+ZSWpF0W1l0SphmKlGoqUakiX3Up3GjPSccyb0E1UHnJs8xkJrqa2iwTg1dWXnirA16fiTqZ1QxSerxskTvAkhEh6m6+r3VW1aw5280quf6M4n36lTNRhkyVmPFEiKf2zV+u/q3buoUUog+RHyZ5+ZjzhZKbCSgSuNvMrKXWaJJU285V1Fd2QSoKOe/Sayw+jg/pKnCuZqnE0cTxUjXlp8lO12623Z+XLi084uwnJB+cAwFICsImTnEaNRuosXZE6S1ekzdJ1VKOKXqZKdZso1W2sVLeRUt3qslfpTnxGd+QbVKbG9SOBhsWGEi/0xPOWlLVq7rzlVUZc4TR2Q7X8df/pdr3/1OxtXsbpuP0SMYOPIkmBm2N27NNFvVk20qj9UlIztqLy6xkFOLh01ZFfY6grk6gTwqpUi7JeaqcxVKpFyPOkDyCHQpImbmUMEYB5VTVZQFpKZBFKeZV0hdKktEoTAlolyqWcRRMCWjHdqVJAJ0oBHSsFdKQU0KFKQOvOQkab6tdoa8MYb4lSUfBOO+e7KKQobrWFDTUl14xUo2WyJBydi97eNhJze7h4jQQd3nqokgxV1ZcIXGHrl6HqKhjq3KQqYStkIpIbCpmI5LcWsgXJDYX0Ph6PjdoeVWz8XtypBKOU12ZC3KkEo5SmxisYpTvOhxwSpUyNlTI1UsrUUCVTdQav0eZrNdpAh9FWwTQSVU5sKCVDERtKCVnFhlIRFLGhVLhSbFgKYk0e8MCVa35xpa5Fpy4BoJJrQooRv7hSyTUhyYjcUEggIjcUknx4/MtqaaWSa1I+jglppZJrUl4Mr3l2nrSKldIqUkqrUCWtdNaf0Zl/gxTSbAotRMlIjB+/FNJsCilzpF8KadaHVBLQL4VUmyJQyyshCYVfDKlLh6nklbIgfSeGVPJKyN/gF0MqeSXkZ/CHuXSGk1oKqW4nZUaYkEIqeSUlPvDKK2lilPLqPFEXKUVdqBJ1OqPM6KyyQYRpNpQWgmK0JAijTWJgpNR5fhGm2VBSqTq/CFNtqEAt7CT+v1+GqYSdQP/3yzCVsBNY+X4ZphJ2Av/dL8PUoXq1r0tnTalFmKqlRAyfEGHa09t5Iiw6X4SFKhGms9SMzlTrRZNKhmnhC0aLKDdagrbRZgkzUqJvv2jSlmZVybBONKmmRuJd+0WTTogJZGq/bFIJMYFI7ZdNKiEmEJr9skklxARSsl82qf1SOvmkFk2q20kUWbmlRIDtWv7yeXFT3dcfql3z70bc7C//fLirntoPP/efPj091JffPd4f79v+Qw7cls/cAs3qfTtJxwB8s43TIZL+4Xg9RdvbJ32/ulytwlVWx5WJirg2yTJYmiq/zkyaJXldpvV1VSaf/z++hqaivXACAA==  
</details>

--- a/core/src/com/unciv/logic/trade/TradeOffer.kt
+++ b/core/src/com/unciv/logic/trade/TradeOffer.kt
@@ -12,12 +12,17 @@ data class TradeOffer(val name:String, val type:TradeType, var amount:Int = 1, v
     init {
         // Duration needs to be part of the variables defined in the primary constructor, 
         // so that it will be copied over with the automatically generated copy()
-        val gameSpeed = UncivGame.Current.gameInfo.gameParameters.gameSpeed
-        duration = when {
-                type.isImmediate -> -1     // -1 for offers that are immediate (e.g. gold transfer)
-                name == Constants.peaceTreaty -> 10
-                gameSpeed == GameSpeed.Quick -> 25
-                else -> (30 * gameSpeed.modifier).toInt()
+        
+        duration =
+            if (type.isImmediate) -1 // -1 for offers that are immediate (e.g. gold transfer)
+            else {
+                // Do *not* access UncivGame.Current.gameInfo in the default constructor!
+                val gameSpeed = UncivGame.Current.gameInfo.gameParameters.gameSpeed
+                when {
+                    name == Constants.peaceTreaty -> 10
+                    gameSpeed == GameSpeed.Quick -> 25
+                    else -> (30 * gameSpeed.modifier).toInt()
+                }
             }
     }
 

